org.alloytools.alloy.application/src/main/java/templates/VelocityTemplateFile.vm

```diff
@@ -1,6 +1,80 @@
+# TODO: add some comments?
```

Comment:
Do we need to put TODO's like this into the code? If there's a bug or performance bottleneck we need to check out later, I am fine with leaving a TODO comment there. I feel like a comment saying to add more comments is not necessary.